models.py

```diff
@@ -1,47 +1,79 @@
-from django.db import models
+import secrets
 
-# Create your models here.
 from django.contrib.auth.models import User
+from django.db import models
+
+CATEGORIES = ((1, 'Trinken'), (2, 'Snacks'), (3, 'Eis'))
+
+
+def generate_key():
+    while True:
+        key = secrets.token_hex(32)
+        if not Purchase.objects.filter(key=key).exists():
+            break
+
+    return key
 
-CATEGORIES = (
-		('D', 'Trinken'),
-		('F', 'Snacks'),
-		('I', 'Eis'),
-	)
 
 class Product(models.Model):
-	product_name = models.CharField(max_length=200)
-	product_price = models.FloatField()
-	product_categorie = models.CharField(max_length=1, choices=CATEGORIES, default="D")
-	product_active = models.BooleanField(default=True)
+    class Meta:
+        ordering = ['name']
+
+    name = models.CharField(max_length=200)
+    price = models.FloatField()
+    category = models.IntegerField(choices=CATEGORIES)
+    active = models.BooleanField(default=True)
```

Comment:
Will changing the name break existing models? Or can it be migrated?

Reply:
Described in the new migrations push.